### PR TITLE
Added "Back To top" button

### PR DESCRIPTION
**Pull request by PrajwalGraj**
Added "Back To top" button in the bottom end of the Readme.md file.

- [ 👾] This pull request is on the dev branch.

---

### CODE REVIEW

**Comment by PrajwalGraj** [comment]
I have read the CLA Document and I hereby sign the CLA


recheck

**Comment by PrajwalGraj** [comment]
@omerzi Please review this PR

**Comment by eyalbe4** [comment]
@PrajwalGraj,
I believe your CLA signing comment is -
```
I have read the CLA Document and I hereby sign the CLA


recheck
```
instead of just being -
```
I have read the CLA Document and I hereby sign the CLA
```
Will you be able to try this?

**Comment by PrajwalGraj** [comment]
I have read the CLA Document and I hereby sign the CLA

File: README.md [hunk 1]
```diff
@@ -294,3 +295,4 @@ Please help us improve Frogbot by [reporting issues](https://github.com/jfrog/fr
 ## 💻 Contributions
 
 We welcome pull requests from the community. To help us improve this project, please read our [Contribution](./CONTRIBUTING.md#-guidelines) guide.
+<p align="right">(<a href="#top">🔼 Back to top</a>)</p>
```
Comment (eyalbe4) [inline README.md:298]: I think this link will look better without the enclosing parentheses.

**Review by eyalbe4 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**